profiling/src/php_ffi.c

```diff
@@ -23,6 +27,37 @@ static ddtrace_profiling_context noop_get_profiling_context(void) {
     return (ddtrace_profiling_context){0, 0};
 }
 
+#if CFG_PRELOAD // defined by build.rs
+static atomic_bool _is_post_startup = false;
```

Comment:
Is there a specific reason for atomics here? This is minit-stage, during which nothing must ever be parallel.

Reply:
I edited the original PR description, but it was maybe after you already looked:
> This isn't strictly necessary in the current state of the PR. In a previous state, it tried accessing the atomic global directly from Rust, which would have made this necessary. However, bindgen was choking on atomic_bool and _Atomic(_Bool) so I switched it. I didn't see a reason to undo this aspect, in case bindgen improves in the future.

Reply:
Yeah, hadn't seen that. I get that rust needs it for safe code, but I prefer at least not declaring something as atomic which may never need atomic semantics.
But I won't insist.

Reply:
Eh, it was easy enough to get rid of.